system/doc/reference_manual/expressions.xml

```diff
@@ -1348,6 +1348,27 @@ handle_call(change, From, #{ state := start } = S) ->
   <section>
     <marker id="bit_syntax"></marker>
     <title>Bit Syntax Expressions</title>
+    <p>The bit syntax operates on <em>bit strings</em>.</p>
+    <p>
+      A <em>bit string</em> is a sequence of bits stored in
+      a sequence of (8-bit) bytes in memory address order.
+      The bits are stored in big-endian format in the byte sequence
+      so the first bit is the higest bit of the first byte.
+      If the length of the <em>bit string</em> is not a multiple
+      of 8 (bits) then the lowest bits of the last byte are unused.
+      A <em>bit string</em> with a length that is a multiple of 8 (bits)
```

Comment:
I think it's better with "8 bits" without the parentheses.

Reply:
Will do